cockatrice/src/settings/gamefilterssettings.cpp

```diff
@@ -92,15 +93,15 @@ int GameFiltersSettings::getMaxPlayers()
     return previous == QVariant() ? 99 : previous.toInt();
 }
 
-void GameFiltersSettings::setMaxGameAgeSeconds(int maxGameAgeSeconds)
+void GameFiltersSettings::setMaxGameAge(const QTime &maxGameAge)
 {
-    setValue(maxGameAgeSeconds, "max_game_age_seconds", "filter_games");
+    setValue(maxGameAge, "max_game_age_time", "filter_games");
 }
 
-int GameFiltersSettings::getMaxGameAgeSeconds()
+QTime GameFiltersSettings::getMaxGameAge()
 {
-    QVariant previous = getValue("max_game_age_seconds", "filter_games");
-    return previous == QVariant() ? -1 : previous.toInt();
+    QVariant previous = getValue("max_game_age_time", "filter_games");
+    return previous.toTime();
```

Comment:
this defaults to an invalid QTime() if the value doesn't match, so no harm in always using toTime()